app/views/shared/_flashes.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if flash.any? %>
```

Comment:
I feel like the `.any?` is redundant with the `.to_hash.slice` below? Since an empty hash would just no-op? I see it's the same as before

Reply:
good point, removed